cmd/validate/image.go

```diff
@@ -487,6 +490,12 @@ func validateImageCmd(validate imageValidationFunc) *cobra.Command {
 		violations, include the title and the description of the failed policy
 		rule.`))
 
+	cmd.Flags().BoolVar(&data.noColor, "no-color", data.info, hd.Doc(`
+		Disable color when using text output even when the current terminal supports it`))
+
+	cmd.Flags().BoolVar(&data.forceColor, "force-color", data.info, hd.Doc(`
```

Comment:
Perhaps just `--color`?
```suggestion
	cmd.Flags().BoolVar(&data.forceColor, "color", data.info, hd.Doc(`
```